src/nature/core.cljc

```diff
@@ -26,8 +26,11 @@
                (fn? fitness-function))]}
    (let [solutions (max 1 (:solutions options))
          carry-over (max 1 (:carry-over options))
-         monitors (:monitors options)]
-     (loop [population (io/build-population population-size allele-set genome-length fitness-function)
+         monitors (:monitors options)
+         sequence-generator-function (:generator options)]
```

Comment:
Could you add this to the docstring for this function?

Reply:
The new commit addresses this, you can find my detailed response below.